tests/platform_tests/test_reload_config.py

```diff
@@ -157,9 +165,14 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     result, out = execute_config_reload_cmd(duthost)
     # config reload command shouldn't work immediately after system reboot
     assert result and "Retry later" in out['stdout']
-
     assert wait_until(300, 20, 0, config_system_checks_passed, duthost, delayed_services)
 
+    # Check if all containers have started
+    assert wait_until(300, 10, 0, check_docker_status, duthost)
```

Comment:
I see multiple reloads and `config_system_checks_passed`. Is the `check_docker_status` required only in the first instance and not in the second to fourth?

Reply:
The logic here is 

- Do cold reboot L#156
- Do config reload and check immediately. Here, as system is not stable because of the previous reboot, so config reload command shouldn't work immediately.  L#165
- Wait for some time to ensure that the system is stable after reboot. L#168 L#171 L#174
- Do config reload again. At this time, the system is stable enough after reboot, so the config reload command works immediately. 

There is no reboot further more.

Reply:
Thank you @yutongzhang-microsoft for the explanation!!